### ISSUE TITLE
Don't abuse debrand_string()

### ISSUE THREAD

**Issue by Bischoff** [SUSE/spacewalk-testsuite-base#259]
function debrand_string() in support/env.rb contains things like

```
        when /Errata .* created./ then str.sub(/Errata/, 'Patch')
        when /.*errata update.*/ then str.gsub(/errata update/, 'patch update')
        when /.*Erratum.*/ then str.gsub(/Erratum/, 'Patch')
        when /.*erratum.*/ then str.gsub(/erratum/, 'patch')
        when /.*Errata.*/ then str.gsub(/Errata/, 'Patches')
        when /.*errata.*/ then str.gsub(/errata/, 'patches')
```

This has the effect of making the test suite cucumber code not match the reality of what is seen on the screen.

Fix the strings directly in the cucumber code.

**Comment by mbologna** [comment]
I'll second this issue. 
We need to uniform all the codebase to one version of naming erratas and patches.

**Comment by renner** [comment]
@Bischoff and @mbologna: The original idea behind `debrand_string()` was to make this testsuite code work for upstream spacewalk as well as for SUSE Manager. With such a function in place we had the option of *debranding* strings only in case we are testing SUSE Manager. Nowadays we can of course rethink this idea as our testsuite is surely quite specific to SUSE Manager anyways. So feel free to pull request a patch that cleans this up or alternatively comes with a better solution to support both targets!

**Comment by MalloZup** [comment]
this is fixed by https://github.com/SUSE/spacewalk-testsuite-base/pull/351/files

We will backport to 31 and head branches. 30 is not compatible with head and 31

Imho we don't need to do that on 30, since not to much active development is there.

If you feel the need, add an issue for fixing in 30

**Comment by MalloZup** [comment]
closing since we fixed for 31/head. for 30 feel free to open an new issue

**Comment by Bischoff** [comment]
Not fixed at all in 3.1 nor head.

debrand_string() is still there and still for example changes "Errata" to "Patches", making the cucumber steps not match the user interface.

**Comment by MalloZup** [comment]
@Bischoff  this issue is fixed in "head-idempotent", that will be the 3.1 and head.

It's just a matter of time.

**Comment by Bischoff** [comment]
OK. Yes, please keep it open until it is merged. As you probably guessed, I checked by doing a "git pull" on head ;-) .
